### DISCUSSION TITLE
String interpolation with {..} give strange result

### DISCUSSION THREAD

**Discussion by magol** [dotnet/csharplang#5919] [General] [answered]
I accidently write following interpolation string which gave a strange result
```C#
Console.WriteLine($"Content of {..} is strange");
```

The output is 

> Content of 0..^0 is strange

I cannot see that this is what it should do. It should be a compilation error.

See https://dotnetfiddle.net/hueM2T

**Comment by HaloFour** [comment, marked as answer]
The syntax `..` is shorthand for an unbounded range or `Range.All`.

**Reply by magol** [reply]
Ok. So I guess that's how it's supposed to be.
